apps/dashboard/test/application_system_test_case.rb

```diff
@@ -2,11 +2,18 @@
 
 class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
 
+  DOWNLOAD_DIRECTORY = Rails.root.join('tmp', 'downloads')
+
   driven_by :selenium, using: :headless_chrome, screen_size: [1400, 1400] do |options|
     # only chrome has support for browser logs
     options.logging_prefs = { browser: 'ALL' }
     options.add_argument('--headless=new')
     options.browser_version = 'stable'
+
+    profile = Selenium::WebDriver::Chrome::Profile.new
+    profile['download.default_directory'] = DOWNLOAD_DIRECTORY
+
+    options.profile = profile
```

Comment:
The new default directory is `~/Downloads`. As that's a bit of a linux thing, this specifies the download directory to be relative to the rails root.